src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -392,8 +380,8 @@ private void validateCloudSdkVersion() {
         throw new CloudSdkOutOfDateException("Cloud SDK version " + version
             + " is too old. Please update to at least " + MINIMUM_VERSION);
       }
-    } catch (ProcessRunnerException ex) {
-      throw new CloudSdkNotFoundException(ex);              
+    } catch (ProcessRunnerException | IllegalArgumentException ex) {
```

Comment:
gcloud returning an empty string instead of a parseable version number. Not sure why it sometimes does this--it's flaky--but we've been seeing this in Eclipse tests.